Tests/fixtures/smartphone-36.yml

```diff
@@ -5568,24 +5552,6 @@
     model: A350
   os_family: Android
   browser_family: Chrome
--
-  user_agent: Mozilla/5.0 (Linux; Android 2.3.6; A350 TV Build/GRK39F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3379.62 Mobile Safari/537.36
-  os:
-    name: Android
-    version: 2.3.6
-    platform: ""
-  client:
-    type: browser
-    name: Chrome Mobile
-    version: 59.0.3379.62
-    engine: Blink
-    engine_version: 59.0.3379.62
-  device:
-    type: tablet
-    brand: Explay
-    model: A350TV
-  os_family: Android
-  browser_family: Chrome
```

Comment:
https://market.yandex.ru/product--smartfon-explay-a350tv/9383578

the smartphone